internal/handlers/pg/pgdb/pool_test.go

```diff
@@ -187,3 +187,82 @@ func TestConcurrentCreate(t *testing.T) {
 		}
 	}
 }
+
+func TestPool_TableExists(t *testing.T) {
```

Comment:
We don't use underscores in test names. We also don't create tests for each function if several functions could be tested together. I think this test the and the following one could be combined together with several subtests

Reply:
Fixed